app/products/calls/screens/call_screen/call_screen.tsx

```diff
@@ -245,9 +251,44 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
         color: 'white',
         margin: 3,
     },
+    errorWrapper: {
+        display: 'flex',
+        width: '100%',
+        padding: 10,
+        paddingTop: 0,
+    },
+    errorBar: {
+        flexDirection: 'row',
+        backgroundColor: theme.dndIndicator,
+        height: CALL_ERROR_BAR_HEIGHT,
+        borderRadius: 5,
+        padding: 10,
+        alignItems: 'center',
+    },
+    errorText: {
+        flex: 1,
+        ...typography('Body', 100, 'SemiBold'),
+        color: '#ffffff',
```

Comment:
same as the other component, probably you want to use `theme.errorTextColor` as well as in the `errorIcon` style

Reply:
I'm using `buttonColor` now, I think that fits best.